Userland/Libraries/LibGfx/Font/OpenType/Font.h

```diff
@@ -138,6 +138,11 @@ class Font : public Gfx::VectorFont {
 
     mutable HashMap<u32, i16> m_kerning_cache;
 
+    mutable String m_family;
+    mutable Optional<u16> m_width;
```

Comment:
nit: use east mutable

Reply:
sure, done. although now it's not consistent with style of other declaration in this class 😅

Reply:
Could add a 2nd commit to move those over I s'pose

Reply:
done

Reply:
@nico  I'm curious, what's the reason for east-`mutable`? I haven't seen that before, and AFAIK the rules that make east-`const` make sense don't apply to `mutable`

Reply:
Just consistency with east const. I hadn't seen a reasoning for that that makes sense, so I was just following the hobgoblin of my little mind. Apologies if we do east const but west mutable! (I added a bunch of east mutable in my code too.) If the serenity style guide doesn't say this yet, maybe it should.

Reply:
It's a little weird, but a general rule of thumb when translating a variable declaration to English is to read it right-to-left, especially when pointers/references are involved:
```c++
int* foo; // foo is a pointer to an int
```

east-`const` ends up making more sense that way, and is more consistent than west-`const`, because the `const` is always to the right of the thing it applies to:
```c++
int* const foo; // foo is a constant pointer to an int
int const* foo; // foo is a pointer to a constant int
int const* const foo; // foo is a constant pointer to a constant int
```

Whereas with west-`const`, the `const` is sometimes to the left and sometimes to the right (because we don't have a choice where the `const`-that-applies-to-the-pointer goes):
```c++
const int* const foo; // first const is to the left, second to the right
```

Thinking about it, some of that applies to `mutable`, like making sense when reading right-to-left ("foo is a mutable int"), so I don't actually have a strong opinion on it. But it's less important than for `const`, because mutable can only appear around the underlying type, i.e. this is illegal:
```c++
int mutable* mutable foo;
```

I think my brain has always read `mutable` as some sort of metadata because, unlike `const`, it isn't part of the type (i.e. the below assertion holds true), so I just always put it on the left.
```c++
int const* foo;
mutable int const* bar;
static_assert(IsSame<decltype(foo), decltype(bar)>);
```